app/controllers/photos_controller.rb

```diff
@@ -50,21 +52,35 @@ def update
 
   # DELETE /photos/1 or /photos/1.json
   def destroy
-    @photo.destroy
-    respond_to do |format|
-      format.html { redirect_back fallback_location: root_url, notice: "Photo was successfully destroyed." }
-      format.json { head :no_content }
+
+    if current_user == @photo.owner
```

Comment:
Good job making sure that the current user is the photo owner before allowing the user to delete!